### ISSUE TITLE
I've always been talking to an audience

### ISSUE THREAD

**Issue by ogbigboss** [ogbigboss/a-wrinkle-in-rhythm#2]
# I don't actually nor have I ever "talked to myself"
#### Maybe now you’ll understand?
2017 - Twitch Streaming
Somewhere in 2017 I started Twitch Streaming - for those that don’t know what this is, essentially you share everything that’s happening on your computer screen as you’re doing it with a live audience. This is most popularly used to show yourself playing computer games.

However, I’m not good at playing video games. I mean, I’m ok. Nothing special.

But in 2017, I wanted to finally start applying what had been then a decade of professional computer programming experience towards a childhood dream: making video games.

I noticed people were streaming themselves making video games - albeit to much smaller audiences than those playing them. I didn’t care about the audience numbers - I just wanted something to keep me consistently learning, so I started streaming game development in 2017.

Quickly I noticed one thing other streamers did, which didn’t change between those playing games and making games - they’d keep a running monologue of what they were doing. Sometimes this would be a quasi dialogue in response or tandem with those typing in “chat” messages for the streamer to engage with.

But sometimes, it was just the streamer creating voice audio, mostly to keep the audience themselves entertained and engaged not just with the game, but with the streamer themselves.

So in 2017, I began to narrate myself while writing code on my livestream. I noticed a few things, mainly that I was quicker in evaluating and realizing mistakes in my code this way. It was like I was constantly reviewing my own decisions in real time. Every now and then, someone in chat helped as well, at the very least with providing their perspectives if not the correct answer to whatever small problem I was trying to solve.

I’d like to think I helped those that were newer to coding in general learn from such a monologue. I’m pretty sure that I’ve been told that I have!

Eventually, on professional gigs where no one was listening to me, I’d do the same. It became a way to get ahead of problems in code before they’d be problems for the companies I worked for.

In 2020, during the pandemic, I worked a job where they were listening to me. Perhaps you’ve heard about it.

#### We all know it was a real investigation to allow this fake reality tv show
So in 2022, this “habit” of mine led to a federal investigation, which led to a (hopeful, since pivoted) reality TV show about it. I’ll outline what that was about later on.

I was, and am, afraid of a situation where I am being monitored and recorded without my consent, in an effort to prove my criminality which doesn’t exist, when - wait for it, drumroll please - on the other side of the ball is the reputation of rich and powerful people.

I’m scared. Literally, right now - I’m scared. This is a reaction born of survivalist instinct.

That said, I know rich and powerful people aren’t scary the way we fantasize them. The “big boss” at the end of most videogames is indeed bigger in size as an individual, and stronger in power as an individual, than the other individual monsters you face along the way.

Reality isn’t like that. The rich and the powerful are only that way because they have human resources, often acquired via economic resources they have cached (look it up).

So I knew that on the other side of what I felt was just a bogus Federal Investigation forced by rich and powerful political and “tech” actors were a bunch of people, well, like me - “middle class” folks working a job for rich and powerful people.

Realizing I got into this mess by talking aloud to a laptop given to me by my workplace to “work from home” with, I began to monologue my entire life as the origins of Twitch dot TV - a website called Justin dot TV - began happening to me, just without my consent.

Justin was a man who decided to live stream every part of his life. It was the origin of Twitch.

I knew I couldn’t convince the rich and powerful of that which they have deployed resources against - I had worked in tech long enough. Once the money’s spent, the resource manager’s - your boss’ - reputation must be defended.

However - I had worked in tech long enough. I knew that in these situations, when whatever bubble “pops” - the “middle class” folks working for the rich and powerful handle things.

I knew the bubble would eventually pop because I’m not a criminal.

Preserving my future meant keeping the “middle class” folks under the employ of the rich and powerful on my side.

So this time when I monologued, I leaned into one thing I maybe held back on a little in Twitch streams, because I don’t care about what a bunch of people in a chat log can respond to about my “political” or “media” views. This time around, there was no chat - I didn’t have to care.

I just had to be funny to make sure that this bogus federal investigation wouldn’t fuck me over.

I didn’t realize it was a Reality TV show. Had I, I would have never engaged this way because of why I didn’t on Twitch - I really don’t care about what anyone, in any class, cares about my thoughts.

But I knew I could make them laugh either way.
I’ve always been talking to an audience
This is why “tricking” me via mental triggers won’t work
I’m not actually mentally ill to the point where I can’t “control” my monologue / dialogue.

I’ve been using it to control my future.

Apparently, it’s made me quite famous?

[I mean, I can] talk to myself for hours [now]. [I can] draw my name in the sand[, even]. [Yes,] I can buy my own flowers.

Any chance you’re understanding why none of what you’ve tried has worked?

#### Was Bickle Bin Laden really just a lie? Or my true self hidin’?
It was a lie. I thought I was under a bogus federal investigation, not a bogus federal investigation turned reality TV show.

If I acted like all this made me commit to “right wing terrorism,” all while showing an ignorance of what that could mean by writing stupid essays including incorrect references to Thomas Sowell, my thought was an actual federal investigation would react by stopping all the corny games and bullshit with tech companies, and just watching me from a distance.

I knew I wasn’t “Bickle Bin Laden”, and was hoping that professing as such would get you all to just watch and wait for the criminal behavior. As it never came over the months, and then months to years - you’d realize I was just lying to get you people to stop acting so fucking corny.

Again, I thought it was just a bogus federal investigation.
But what about all your crazy behavior once you realized it was a Reality TV Show?
I’ve been trying to “get your dumb little show canceled”, to quote me
So in July 2022, when I realized that yes, this is and probably was a “Reality TV” show, rather than investigate it and try to find out what you wanted out of it, I figure’d canceling it was better for me, the country, and, well, actual “Reality” as it were.

So if I felt like it was a dating show? I’d act like an undateable man.

Are you trying to scare me into thinking “people are after me?” I’d chastise you for acting inappropriately under the guise of the Federal authority you were given.

If you tried painting me as some criminal hacker mastermind? I’d start talking about shit like magic being real and being a witch or something ridiculous like that.

If you kept coming after me in all manner or stupid shit, including using Bruce Springsteen, Jann Wenner, and Paula Patton et al and not in that order? I’d start letting you know how I really feel about art and “getting it,” knowing that if you used these artists against me, you didn’t “get” art.

If you do shit like lighting a truck on fire across the street from my apartment like the KKK never did the same shit to black people in this country? I’d start acting mentally ill to see if suicidality is something you’d have any compassion for and thus cancel your show.

If you saw mental illness and suicidality as an advantage such taht you ignored signs and you kept following me around Manhattan even when I desperately just wanted a night alone in a hotel I shouldn’t of had to pay for, only to somehow dipshittedly go to the one hotel I probably should’ve of avoided? Well, then I’d start talking to psychics and strippers. Hey, Sophias!

Whatever you wanted to portray me as, I’d go in another direction, hoping you’d burn your budget and leave me alone.

One thing is the honest to god’s truth - I’m an incredible liar. Paint me as such in a way that wasn’t true, but would help you? Well, you’d have to be better at lying than me. And then your return on investment might work. Maybe.

But that said, you weren’t here to make a profit - you were here for something else.
Before the denouement, some catharsis
I find this incredibly insulting and you should (have) as well.
You’ve been telling scores of people that I’m so crazy that simply walking by me or talking out loud near me will “trigger” psychosis or some shit - which is actually good for you! Somehow?

Morals, much?

I find the notion that so many people find me to be mentally weak to that extent insulting. I find the idea that there would be any sort of social or economic advantage to proving this about me, even when it’s false, to be incredibly insulting. You should too.

[All that said, it’s not best to] get mad [if you can] get even. [So,] I don’t talk shit [to myself] - I just flip it on you. I’m a thief in the house of love, and I can’t be trusted.

#### This has nothing to do with capitalism, communism, or any system, yet this human flaw affects all of them.

Social relationships matter more than economic or business sense for the “resource management” class, and this is the same complaint people make about both “communism” and “capitalism” - those with control of resources over society will use them to deepen their control over society itself, invariably, and almost akin to a third way of phrasing a “theory of motion.”

If I was wrong, and you were “business people” - you would’ve walked away from this sunken cost fallacy long ago. Years ago!

Your client relationships matter more for investment cash flow than business cash flow. Too many of your livelihoods are made on the social relationships that incur investment, at the expense of the “resource management” class’ knowledge of how business actually works.

In other words? You haven’t even made a product here! Anything you make will surely be a loss compared to your avenues to exploit them for revenue against the expenses you’ve incurred over these past 18 months (or more) and counting!

But if you can succeed, then maybe there’ll be future investment… but only if you make, oh bummer, not just anyone, but someone else happy.

About something that isn’t real - I’m not actually talking to myself.

Morality be damned! Christianity, surely too, how else do we know about the word damned!

So, then – who am I talking to?

Care to introduce yourselves?

I’ll wait.

**Comment by ogbigboss** [comment]
UPDATE: I'm gonna give this schtick up; at least for the next stretch of this TV show we've all been producing that I've been acting in for us.

If you're familiar with writing code on Github, and using Pull Requests, you may say - shouldn't I close the issue then?

Well, familiarity with Github being unnecessary, given that this epic journey of ours has mostly been driven by the "wind in my throat", knowing if you held that against me you'd have to provide a decent answer as to _why_ you were recording me in the first place, and the litany of fun that inadmissibility has created me given the grey area of what could be defined as a _trick_ as my awareness of your surveillance denies that what you are recording may not necessarily be _true_, similar to how law enforcement sets up criminals - well, if `entraptertainment` is so fun and cool and cop hip, then "why give it up?"

And the answer to both questions is this: Because it's of no current utility to me.

It's been fun, though, to those that brought this show back to my childhood home in Montclair.

Now, I'm sure I'll slip up here and there. After all, who doesn't mutter something to themselves from time to time?

I'll miss quoting GK Chesterton, Mary Karr, and Maye Musk to you all from my bedroom window.

It's probably for the best as I approach Marcus Aurelius' _Meditations_ and Vyasa's _Bhagavad Gita_.

God(s) would only know what you'd think of me then!

**Comment by ogbigboss** [comment]
oh but I'm still gonna say "excuse me" after burping most likely - it just feels right. I'm keeping that one.
